ow_sentry_utils/__init__.py

```diff
@@ -1,3 +1,14 @@
+import re
+
+
+SENTRY_IGNORE_ERRORS = {
```

Comment:
what about making this a configurable setting? So it can be overridden in each installation if needed.

Would be good to have a deafult value, eg:

```python
DEFAULT_SENTRY_IGNORE_ERRORS = {
# ...
}

SENTRY_IGNORE_ERRORS = getattr(settings, DEFAULT_SENTRY_IGNORE_ERRORS)
```

So in each installation we could do:

```
from ow_sentry_utils import DEFAULT_SENTRY_IGNORE_ERRORS

SENTRY_IGNORE_ERRORS = deepcopy(DEFAULT_SENTRY_IGNORE_ERRORS)
SENTRY_IGNORE_ERRORS.update({
    # edits...
})
```

What do you think?

And will be good to document this briefly in the README too.

Reply:
This will make this module dependent on Django.

We can also update the setting like this 

```python
# in settings.py of the project
from ow_sentry_utils import SENTRY_IGNORE_ERRORS, before_send

# Make changes to the dictionary 

SENTRY_IGNORE_ERRORS.update({
    ...
})


sentry_sdk.init(
   ...
   before_send=before_send,
)

```

Reply:
yeah ok, I thought we would use this in django only but we can use it also in the wireguard flask app :+1: